pkg/httpserver/alertmanagement.go

```diff
@@ -0,0 +1,108 @@
+package httpserver
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"k8s.io/klog/v2"
+
+	alertmanagement "github.com/openshift/cluster-monitoring-operator/pkg/alert/management"
+)
+
+type alertManagementMux struct {
+	alertsManagementController alertmanagement.Controller
+}
+
+func AlertManagementMux(alertsManagementController alertmanagement.Controller) *http.ServeMux {
+	mux := http.NewServeMux()
+
+	amm := &alertManagementMux{
+		alertsManagementController: alertsManagementController,
+	}
+
+	// CRUD operations for alerting rules
+	mux.HandleFunc("GET /rules", amm.listAlertingRulesHandler)
+	mux.HandleFunc("POST /rules", amm.createAlertingRuleHandler)
+	mux.HandleFunc("PUT /rules/{alertname}", amm.updateAlertingRuleHandler)
+	mux.HandleFunc("DELETE /rules/{alertname}", amm.deleteAlertingRuleHandler)
+
+	// Enable/Disable operations for alerting rules
+	mux.HandleFunc("POST /rules/{alertname}/disable", amm.disableAlertingRuleHandler)
```

Comment:
Since we can have the same alert with different severity, I think it would be better to also include the severity in the API.

Reply:
nit:
it will be something like this (w/o the additional `/`, but with `&` instead):
```
/alerting/rules/{alertname}?severity={severity}&enable?value=true/false
```

Reply:
but the 2nd one is less RESTy and may be confusing.

Reply:
its already as:

POST /rules/{alertname}/disable?severity={severity}
POST /rules/{alertname}/enable?severity={severity}

severity is optional
as we can drop the alertname for all severities

should become more obvious when we generate documentation at the end

is it not okay?